touki.perf/touki.perf.csproj

```diff
@@ -27,7 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\touki.tests\touki.tests.csproj" />
+    <ProjectReference Include="..\touki.tests\touki.tests.csproj" ReferenceOutputAssembly="false" />
```

Comment:
You can't do this it needs the reference. Set the entry point explicitly to fix the underlying problem.